src/Button/ZoomToExtentButton/ZoomToExtentButton.tsx

```diff
@@ -125,14 +127,14 @@ class ZoomToExtentButton extends React.Component<ZoomToExtentButtonProps> {
       ...fitOptions
     };
 
-    if (extent && (center && zoom)) {
+    if (extent && (center && _isFinite(zoom))) {
       logger.warn('Provide either an extent or a center and a zoom.' +
```

Comment:
I think the warning message is still somewhat unclear. At this point we already know that both extent and center/zoom are provided

So something like `Both extent and center / zoom are provided. Extent will be used in favor of center / zoom`

Reply:
good point thanks!